Validation in Admin Register page

sankettank66 opened the pull request:
Hello, @MuhammadKhalilzadeh

As per the requirements of the task, I have implemented the following in `/admin-reg`:

- Get the input values
- Implement input validation checks

Let me know if there are any changes needed.

sankettank66 reviewed (commented):
@MuhammadKhalilzadeh

Please review this PR and let me know if there are any changes. If not, I can proceed further.

MuhammadKhalilzadeh commented:
@sankettank66 
Could you please add a screen shot.
How will it look like when validation errors work?

sankettank66 commented:
Hello @MuhammadKhalilzadeh ,

This pull request includes the implementation of various input validations to improve user experience and data integrity.

### Changes Made
1. **Empty Input Validation**: 
   - Added error messages indicating that all fields are required.
  
2. **Password Validation**: 
   - Enforced the following requirements for the password:
     - At least one uppercase letter
     - At least one number
     - At least one special character
     - Minimum length of 8 characters
  
3. **Password Confirmation**: 
   - Implemented a check to ensure that the password and confirm password fields match.

4. **Email Validation**: 
   - Integrated a regex validation for email format.

### Screenshots
- **All Empty Input Errors**:  
  ![AllErrorsValidation](https://github.com/user-attachments/assets/21c0ac66-22c1-4a76-93b9-841cdff6cadb)
  
- **Password Validation**:  
  ![PasswordValidation](https://github.com/user-attachments/assets/7e165e38-e2b4-41e5-94b4-c9c51a704001)
  
- **Password Validation (Extended)**:  
  ![PasswordValidation2](https://github.com/user-attachments/assets/81709e7a-636b-402a-b10b-34e606896756)

MuhammadKhalilzadeh reviewed (approved):
@sankettank66  That's fantastic. Thank you ✌️👍🏻
